saiph/inverse_transform_test.py

```diff
@@ -128,28 +164,61 @@ def test_inverse_from_coord_mca(
     model = fit(wbcd_quali_df, nf="all")
 
     reversed_individuals = inverse_transform(
-        wbcd_supplemental_coord_quali, model, use_max_modalities=False
+        wbcd_supplemental_coord_quali_linux_generated, model, use_max_modalities=False
     )
 
     reversed_individuals = reversed_individuals.astype("int")
     wbcd_quali_df = wbcd_quali_df.astype("int")
 
-    reversed_statistics = reversed_individuals.describe()
-    wbcd_statistics = wbcd_quali_df.describe()
+    to_compare = [
+        CompareDistributions(label="count"),
+        CompareDistributions(label="mean", atol=0.25),
+        CompareDistributions(label="std", atol=0.35),
+        # assert equal for the min as there are many low values
+        CompareDistributions(label="min"),
+        CompareDistributions(label="max"),
+        CompareDistributions(label="25%", atol=1),
+        CompareDistributions(label="50%", atol=1),
+        CompareDistributions(label="75%", atol=2),
```

Comment:
Maybe you can have a df or a dictionary to manage all absolute tolerance of the two tests at the same place.